tf-managed/modules/daily-snapshot/service/init.sh

```diff
@@ -21,5 +21,20 @@ done'
 # Run new_relic and fail2ban scripts
 bash newrelic_fail2ban.sh
 
-# Setup cron jobs
-cp calibnet_cron_job mainnet_cron_job /etc/cron.hourly/
+echo "$SNAPSHOT_TYPE"
+
+# Setup cron job
+IFS=',' read -ra ADDR <<< "$SNAPSHOT_TYPE"
+for type in "${ADDR[@]}"; do
+  case "$type" in
+    mainnet)
+      mv mainnet_cron_job /etc/cron.hourly/
+      ;;
+    calibnet)
+      mv calibnet_cron_job /etc/cron.hourly/
+      ;;
+    *)
+      echo "Error: Invalid network type '$type'"
+      ;;
+  esac
+done
```

Comment:
It can be a follow-up, but we can generate the cron scripts now, instead of copying pre-defined ones.

Reply:
sure sure.